### PR TITLE
chore: Add 24.11 release

### PR DESCRIPTION
**Pull request by Techassi**
Part of https://github.com/stackabletech/issues/issues/647

> [!CAUTION]
> Before merging, the description needs to be set.

### CODE REVIEW

File: releases.yaml [hunk 1]
```diff
@@ -1,9 +1,46 @@
 ---
 releases:
+  24.11:
+    releaseDate: &latest-release-release-date 2024-11-18
+    description: TBD
```
Comment (NickLarsenNZ) [inline releases.yaml:5]: @lfrancke, is there a one-liner description of the release that can go in here?

Comment (lfrancke) [reply releases.yaml:5]: I don't know any...where does that even show?

Comment (Techassi) [reply releases.yaml:5]: In the `stckablectl release list` output.

Comment (lfrancke) [reply releases.yaml:5]: If anyone comes up with anything please add it.. Otherwise I'm fine with just a generic "The November 2024 release." As well.

Comment (NickLarsenNZ) [reply releases.yaml:5]: We should probably link to the release notes in future. But here is the suggestion to apply:

```suggestion
    description: The November 2024 release
```

I believe @Maleware is waiting on this before he can install the new operators during the ARM testing.

**Review by NickLarsenNZ — APPROVED** [review]
LGTM

**Pull request merged**